nestjs/src/profile/dto/update-profile.dto.ts

```diff
@@ -234,3 +234,80 @@ export class ProfileInfoDto {
   @IsBoolean()
   isProfileSettingsChanged: boolean;
 }
+
+export class UpdateProfileInfoDto {
+  @ApiProperty({ required: false })
+  @IsOptional()
+  @IsString()
+  name?: string;
+
+  @ApiProperty({ required: false })
+  @IsOptional()
+  @IsString()
+  githubId?: string;
+
+  @ApiProperty({ required: false, nullable: true, type: String })
+  @IsOptional()
+  aboutMyself?: string | null;
+
+  @IsString()
+  @ApiProperty({ required: false, nullable: true, type: String })
+  @IsOptional()
+  cityName?: string | null;
+
+  @IsString()
+  @ApiProperty({ required: false, nullable: true, type: String })
+  @IsOptional()
+  countryName?: string | null;
+
+  @ApiProperty({ required: false, nullable: true, type: [Education] })
+  @IsOptional()
+  @IsArray()
+  educationHistory?: Education[];
+
+  @ApiProperty({ required: false, nullable: true, type: String })
+  @IsOptional()
+  @IsString()
+  englishLevel?: EnglishLevel | null;
+
+  @ApiProperty({ required: false, nullable: true, type: String })
+  @IsOptional()
+  @IsString()
+  contactsPhone?: string | null;
+
+  @ApiProperty({ required: false, nullable: true, type: String })
+  @IsOptional()
+  @IsString()
+  contactsEmail: string | null;
```

Comment:
should it be `contactsEmail?: string | null;` like everything else?

Reply:
thanks, fixed